src/com/senlainc/warsaw/tyurin/Tank.java

```diff
@@ -0,0 +1,30 @@
+package com.senlainc.warsaw.tyurin;
+
+public class Tank implements IProduct, IAssemblyLine{
```

Comment:
Same as parts but now product, you already should understand what here incorrect